test/test_quantized.py

```diff
@@ -331,11 +331,9 @@ def test_qmul_relu_same_qparams(self):
         mul = torch.ops.quantized.mul_scalar
         for b in B:
             C_ref = qA.dequantize().numpy() * b.item()
-            qC = _quantize(C_ref, scale, zero_point)
-            dqC = _dequantize(qC, scale, zero_point)
-            qC_hat = mul(qA, b.item(), scale, zero_point)
-            dqC_hat = qC_hat.dequantize()
-            self.assertEqual(dqC, dqC_hat)
+            qC_hat = mul(qA, b.item())
+
+            self.assertEqual(C_ref, qC_hat.dequantize())
```

Comment:
This test should not be passing with the existing code for negative b. If it does, then we are missing something. Can you check?